package.json

```diff
@@ -17,7 +17,8 @@
     "url": "https://github.com/wildbit/stylelint-config-wildbit/issues"
   },
   "homepage": "https://github.com/wildbit/stylelint-config-wildbit#readme",
-  "devDependencies": {
-    "stylelint": "^7.1.0"
+  "dependencies": {
+    "stylelint": "^8.0.0",
```

Comment:
As Stylelint adopted a new semantic version policy a use of the tilde (~) in package.json [is now recommended](https://github.com/stylelint/stylelint/issues/1804).